### PR TITLE
Add router

### PR DESCRIPTION
**Pull request by NeversSync**

### CODE REVIEW

File: tea-gallery-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <button>
+            <Link to="/albums"><h3>Albums</h3></Link>
+          </button>
+          <button>
+            <Link to="/"><h3>Home</h3></Link>
+          </button>
+        </div>
+        <Switch>
+          <Route path="/albums" component={Albums} />
+          <Route redirect="/" />;
```
Comment (dchasepdx) [inline tea-gallery-app/src/App.js]: redirect is not a prop you can pass to `<Route />`. There is a specific `<Redirect />` component.

File: tea-gallery-app/src/test/AlbumDetail.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    description: 'what a nice tetsubin',
+    url: '/../public/img/tetsubin-charcoal.jpg'
+  },
+  {
+    title: 'Teapot display',
+    description: 'what a nice teapot display',
+    url: '/../public/img/teapot-display.jpg'
+  }
+];
+
```
Comment (dchasepdx) [inline tea-gallery-app/src/test/AlbumDetail.test.js]: This isn't the best way to test this according to React Router, but I couldn't find an alternative. Create a mock match object like this:
```
const match = {
  params: {
    albumId: '123'
  }
};
```

Comment (NeversSync) [inline tea-gallery-app/src/test/AlbumDetail.test.js]: I don't know what you mean, where would i put that match object? Inside the image object? thanks

File: tea-gallery-app/src/App.js (hunk 1 above)
Comment (NeversSync) [inline tea-gallery-app/src/App.js]: done, thx!

**Pull request merged**